lib/Steps/Integrations/ReactNative.ts

```diff
@@ -132,6 +119,32 @@ export class ReactNative extends MobileProject {
     return result;
   }
 
+  private async _patchJsSentryInit(
+    platform: string,
+    answers: Answers,
+  ): Promise<void> {
+    const prefixGlob = '{.,./src}';
+    const suffixGlob = '@(j|t|cj|mj)s?(x)';
+    const platformGlob = `index.${platform}.${suffixGlob}`;
+    // rm 0.49 introduced an App.js for both platforms
+    const universalGlob = `App.${suffixGlob}`;
+    const jsFileGlob = `${prefixGlob}/+(${platformGlob}|${universalGlob})`;
+
+    const jsFileToPatch = matchFiles(jsFileGlob);
+    if (jsFileToPatch.length !== 0) {
+      await patchMatchingFile(
+        jsFileGlob,
+        this._patchJs.bind(this),
+        answers,
+        platform,
+      );
+      dim(`✅ Patched ${jsFileToPatch.join(', ')} file(s).`);
+    } else {
+      dim(`🚨 Could not find ${platformGlob} nor ${universalGlob} files.`);
+      yellow('❓ Please, visit https://docs.sentry.io/platforms/react-native/#initialize');
```

Comment:
```suggestion
      dim(`🚨 Could not find ${platformGlob} nor ${universalGlob} files.`);
      yellow('❓ Please, visit https://docs.sentry.io/platforms/react-native');
```

Just in case we change the headline, the docs will be broken.